cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Tx.hs

```diff
@@ -139,11 +140,14 @@ data TxRedeemer = TxRedeemer
 
 data TxScript = TxScript
   { txScriptHash :: !ByteString
+  , txScriptType :: ScriptType
   , txScriptPlutusSize :: Maybe Word64
   , txScriptJson :: Maybe ByteString
   , txScriptCBOR :: Maybe ByteString
   }
 
+data ScriptType = MultiSig | Timelock | Plutus
```

Comment:
Why not just use the `ScriptType` defined in `cardano-db/src/Cardano/Db/Types.hs` ?

Reply:
We don't import anything from Cardano.Db in the Generic module, so I thought I'd keep it that way.

Reply:
`cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Generic/Util.hs` has
```
import qualified Cardano.Db as Db
```